src/supertux/moving_object.cpp

```diff
@@ -29,6 +30,16 @@ MovingObject::MovingObject(const ReaderMapping& reader) :
   GameObject(reader),
   m_col(COLGROUP_MOVING, *this)
 {
+  float height, width;
+
+  if (reader.get("width", width))
+    m_col.m_bbox.set_width(width);
+
+  if (reader.get("height", height))
+    m_col.m_bbox.set_height(height);
+
+  reader.get("x", m_col.m_bbox.get_left());
```

Comment:
I'm confused about this. the get_left() and get_top() methods are marked const. However, they're passed by reference to the reader.get() method which might attempt to save a value, which should fail. Can you please test this?

Reply:
@tobbi There is another place in the code that uses the exact same trick, the author called it a "temporary hack" (which I (or anyone) should fix someday). Two overloads for that function, of which one is a non-const reference.
![image](https://user-images.githubusercontent.com/66701383/103181497-9eef8880-486f-11eb-9d11-80f34e5117c1.png)

Reply:
Very interesting. OKay, I'm gonna merge this.